web/wwwroot/css/site.css

```diff
@@ -59,5 +59,10 @@ input.invalid {
 }
 
 .tables{
-  width: 50%;
+  display:inline-block;

```

Comment:
Let's add a space after `:` so our CSS looks consistent

Reply:
And also did you mean to remove that width property? Looks like our table will span the whole page now 🙂